### ISSUE TITLE
pre-commit markdownlint-docker doesn't allow --disable?

### ISSUE THREAD

**Issue by vunhatchuong123** [igorshubovych/markdownlint-cli#374]
I tried

```yaml
-   repo: https://github.com/igorshubovych/markdownlint-cli
    rev: v0.33.0
    hooks:
    -   id: markdownlint-docker
        args: [--disable MD041 -- README.md]
```

output:

```bash
error: unknown option '--disable  MD041 -- README.md'
```

**Comment by vunhatchuong123** [comment]
After more tests, the correct syntax is:

```yaml
-   repo: https://github.com/igorshubovych/markdownlint-cli
    rev: v0.33.0
    hooks:
    -   id: markdownlint-docker
        args: [--disable, MD041, --]
```
